packages/docs/src/pages/en/styles/display.md

```diff
@@ -73,6 +73,20 @@ The _condition_ applies the class base on:
 
 Additionally, **media types** can be targeted using the `only` condition. Both `hidden-screen-only` and `hidden-print-only` are currently supported.
 
+### Caveats
+
+<alert type="info">
+
+It is important to note that using any of the display classes above will result in any display style previously added being overwritten. This is because of the classes using `!important` in their display styling.
+
+#### Example
```

Comment:
The example section can be removes, the previous portion already implies this.

Reply:
This should also be resolved now.